streams/examples/main/java/org/apache/kafka/streams/examples/pipe/PipeJob.java

```diff
@@ -26,6 +26,12 @@
 
 import java.util.Properties;
 
+/**
+ * Implements a simple "Pipe" program that consumes from a source topic "streams-file-input"
```

Comment:
I'd clarify this to:

> Implements a simple "pipe" program that reads from a source topic "streams-file-input" and writes the data as-is (i.e. unmodified) into a sink topic "streams-pipe-output".  Source and sink topic names are specified using the high-level KStream DSL.
>
> Before running this example you must create the source topic (e.g. via `bin/kafka-topics.sh --create ...`) and write some data to it (e.g. via `bin-kafka-console-producer.sh`).  Otherwise you won't see any data arriving in the output topic.